reference/OpenAPI.json

```diff
@@ -17801,8 +17801,8 @@
           }
         ]
       },
-      "RedemptionRedeem": {
```

Comment:
Tutaj to tak nazwałem, bo to chodzi o schemat redempcji tylko dla samego Redeem

Reply:
wiem wiem, ale w ten sposob jest mniej breaking changes w sdk
a wole by ludzie mieli mniej poprawiania po podbiciu wersji.